src/screens/LoginPage/LoginPage.tsx

```diff
@@ -40,6 +40,7 @@ function LoginPage(): JSX.Element {
     email: '',
     password: '',
   });
+  const [show, setShow] = useState(false);
```

Comment:
Since we are using typescript, you should type this e.g `useState<boolean>(false)`